engine/engine/content/builtins/manifests/web/engine_template.html

```diff
@@ -110,6 +110,7 @@ <h1>WebGL not supported ⚠️</h1>
 		}
 	}
 
+	Module['persistentStorage'] = !!(window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB);
```

Comment:
`!!` to convert `IDBFactory` to boolean
for `IDBFactory` it will be `true`
for `null`, `undefined`, 0, `NaN`, '' etc it will be `false`